Well, bummer. Test fails like this:
```
 2024/04/24 13:39:03 WARN + cat out
2024/04/24 13:39:03 INFO failed [1]: java -jar /usr/share/java/wavefront/wavefront-proxy/wavefront-proxy.jar
2024/04/24 13:39:03 INFO Exception in thread "main" java.lang.NoClassDefFoundError: io/netty/handler/codec/http/cors/CorsConfig
2024/04/24 13:39:03 INFO     at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
2024/04/24 13:39:03 INFO     at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3166)
2024/04/24 13:39:03 INFO     at java.base/java.lang.Class.getDeclaredMethod(Class.java:2473)
2024/04/24 13:39:03 INFO     at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:47)
2024/04/24 13:39:03 INFO     at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
2024/04/24 13:39:03 INFO     at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
2024/04/24 13:39:03 INFO     at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65)
2024/04/24 13:39:03 INFO Caused by: java.lang.ClassNotFoundException: io.netty.handler.codec.http.cors.CorsConfig
2024/04/24 13:39:03 INFO     at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
2024/04/24 13:39:03 INFO     at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:594)
2024/04/24 13:39:03 INFO     at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151)
2024/04/24 13:39:03 INFO     at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:527)
2024/04/24 13:39:03 INFO     ... 7 more
2024/04/24 13:39:03 WARN + exit 1
2024/04/24 13:39:03 INFO pod e02a90f14c3452f5ca879b358265b4e48b836169795298c31e38abfa860c28b9 terminated
2024/04/24 13:39:03 INFO ERROR: failed to test package. the test environment has been preserved:
2024/04/24 13:39:03 INFO   workspace dir: /temp/melange-workspace-706052928
```

some local user error made the test pass locally.  I had removed the bump of `netty-codec-http` entirely, and then built the package, seen it work and then saw a 4.1.109 available and thought "oh maybe that fixes it".  I put in that change, but didn't bump the epoch again.

Anyway, the test is good to have, and you can make the test pass simply with this diff:
```diff
diff --git a/wavefront-proxy/proxy/pombump-deps.yaml b/wavefront-proxy/proxy/pombump-deps.yaml
index a3d5de67b..aad2bee73 100644
--- a/wavefront-proxy/proxy/pombump-deps.yaml
+++ b/wavefront-proxy/proxy/pombump-deps.yaml
@@ -17,11 +17,6 @@ patches:
     version: 4.1.109.Final
     scope: import
     type: jar
-  - groupId: io.netty
-    artifactId: netty-codec-http
-    version: 4.1.109.Final
-    scope: import
-    type: jar
   - groupId: com.squareup.okio
     artifactId: okio
     version: 3.4.0
```

--- a/wavefront-proxy.yaml
+++ b/wavefront-proxy.yaml
@@ -1,7 +1,7 @@
 package:
   name: wavefront-proxy
   version: "13.4" # When version is bumped, check if patches are still needed to address CVE-2023-1428
-  epoch: 4
+  epoch: 5
   description: Wavefront Proxy Project
   copyright:
     - license: Apache-2.0

--- a/wavefront-proxy.yaml
+++ b/wavefront-proxy.yaml
@@ -72,6 +72,29 @@ subpackages:
         - wavefront-proxy-config
         - wavefront-proxy-licenses
 
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+  pipeline:
+    - runs: |
+        set -- java -jar /usr/share/java/wavefront/wavefront-proxy/wavefront-proxy.jar
+        "$@" --help >out 2>&1 || {
+          rc=$?;
+          echo "failed [$rc]: $*"
+          cat out
+          exit $rc;
+        }
+        expected="Wavefront Proxy version ${{package.version}}"
+        grep "$expected" out || {
+          echo "command '$* --help' did not contain expected output"
+          echo "expected: $expected"
+          echo "actual:"
+          cat out
+          exit 1
+        }
+
 update:
   enabled: true
   github:

--- a/wavefront-proxy/proxy/pombump-deps.yaml
+++ b/wavefront-proxy/proxy/pombump-deps.yaml
@@ -14,12 +14,7 @@ patches:
   # Fixes GHSA-5jpm-x58v-624v
   - groupId: io.netty
     artifactId: netty-bom
-    version: 4.1.108.Final
-    scope: import
-    type: jar
-  - groupId: io.netty
-    artifactId: netty-codec-http
-    version: 4.1.108.Final
+    version: 4.1.109.Final
     scope: import
     type: jar
   - groupId: com.squareup.okio

